js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AutomaterializeRequestedPartitionsLink.tsx

```diff
@@ -63,15 +66,20 @@ export const AutomaterializeRequestedPartitionsLink = ({runIds, partitionKeys}:
     }
 
     return runIds ? (
-      <PartitionAndRunList runIds={runIds} partitionKeys={filteredPartitionKeys} />
+      <PartitionAndRunList runIds={runIds} partitionKeys={filteredPartitionKeys} intent={intent} />
     ) : (
       <VirtualizedPartitionList partitionKeys={partitionKeys} />
     );
   };
 
   return (
     <>
-      <ButtonLink onClick={() => setIsOpen(true)}>{label}</ButtonLink>
+      <Box flex={{direction: 'row', gap: 8, alignItems: 'center'}}>
+        <Tag intent={intent}>{label}</Tag>
+        <ButtonLink onClick={() => setIsOpen(true)}>
+          <Caption>View details</Caption>
```

Comment:
If you find that the text of this link is (incorrectly) gray, you can probably switch it around:

```
<Caption>
  <ButtonLink>...</ButtonLink>
</Caption>
```

Reply:
its blue 🙂